Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -20,18 +20,18 @@ let headerSeparator: StaticString = ": "
 /// A representation of the request line and header fields of a HTTP request.
 public struct HTTPRequestHead: Equatable {
     /// The HTTP method for this request.
-    public let method: HTTPMethod
+    public var method: HTTPMethod
 
     // Internal representation of the URI.
-    private let rawURI: URI
+    private var rawURI: URI
```

Comment:
This one is private so I think this makes no sense or I am missing something ?

Reply:
Yeah, that one could stay `let`.

Reply:
I missed adding a `set` case to `uri`, that is added now (and relies on rawURI being mutable)